src/Components/Moormanage/Boatyards/Boatyards.tsx

```diff
@@ -1,233 +1,223 @@
-import React, { useEffect, useState } from "react";
-import { DataTable } from "primereact/datatable";
-import { Column } from "primereact/column";
-import CustomModal from "../../customComponent/CustomModal";
-import AddBoatyards from "./AddBoatyards";
-import { InputText } from "primereact/inputtext";
-import MooringTable from "./BoatyardTable";
-import { BOATYARD_DATA, BOATYARD_PAYLOAD, BOATYARD_RESPONSE } from "../../../Services/MoorManage/types";
-import BoatyardTable from "./BoatyardTable";
-import { useGetBoatyardsMutation } from "../../../Services/MoorManage/moormanage";
+import React, { useEffect, useState } from 'react'
+import CustomModal from '../../CustomComponent/CustomModal'
+import AddBoatyards from './AddBoatyards'
+import { InputText } from 'primereact/inputtext'
+import {
+  BoatYard_Data,
+  BoatYard_Payload,
+  BoatYard_Response,
+} from '../../../Type/ApiTypes'
+import BoatyardTable from './BoatyardTable'
+import { useGetBoatyardsMutation } from '../../../Services/MoorManage/MoormanageApi'
 
 const Boatyards = () => {
-  const [modalVisible, setModalVisible] = useState(false);
-  const [boatyardsData, setboatyardsData] = useState<BOATYARD_PAYLOAD[]>([]);
-  const [filteredboatyardsData, setFilteredboatyardsData] = useState<
-    BOATYARD_PAYLOAD[]
-  >([]);
-  const [getBaotyards] = useGetBoatyardsMutation();
+  const [modalVisible, setModalVisible] = useState(false)
+  const [boatyardsData, setboatyardsData] = useState<BoatYard_Payload[]>([])
+  const [filteredboatyardsData, setFilteredboatyardsData] = useState<BoatYard_Payload[]>([])
+  const [getBaotyards] = useGetBoatyardsMutation()
 
-  const moorings: BOATYARD_DATA[] = [
+  const moorings: BoatYard_Data[] = [
```

Comment:
check pascal case spec from here

https://medium.com/@code.ceeker/naming-conventions-camel-case-pascal-case-kebab-case-and-more-dc4e515b9652#:~:text=The%20Pascal%20Snake%20Case%20is,with%20the%20underscore%20%E2%80%9C_%E2%80%9D.&text=This%20convention%20is%20observed%20in%20legacy%20Java%20packages.

Reply:
check again

Reply:
resolved now